src/lib/components/progress-bar.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
```

Comment:
Is this worthy of being in holocene?

Reply:
Ahh I thought it was in Holocene, definitely move it there.